src/mods/UObjectHook.cpp

```diff
@@ -1463,6 +1463,10 @@ sdk::UObject* UObjectHook::StatePath::resolve_base_object() const {
     return nullptr;
 }
 
+// unordered map of previously derived object name / property name matches (key is name in the JSON file,
+// value is object / property name from the game which can contain different numeric characters)
+std::unordered_map<std::string, std::string> cachedLookups;
```

Comment:
should not be a global, but rather a member variable

Reply:
i've moved this to be an inline static member of StatePath and tidied up a little